accord-core/src/main/java/accord/messages/ReadData.java

```diff
@@ -513,10 +513,17 @@ public enum CommitOrReadNack implements ReadData.ReadReply
          * The commit has been rejected due to stale ballot.
          */
         Rejected("CommitRejected"),
+
         /**
          * Either not committed, or not stable
          */
         Insufficient("CommitInsufficient"),
+
+        /**
+         * PreApplied successfully, but the request is blocking so waiting to reply
+         */
+        Waiting("ApplyWaiting"),
```

Comment:
I don't see this sent in this patch, is this done in C*?

Reply:
I don't see it used in https://github.com/apache/cassandra/pull/3698

Reply:
hmm, let me see where it went - it should be here as part of the new sync point logic.

Reply:
It's used in `ApplyThenWaitUntilApplied`